### DISCUSSION TITLE
2k屏幕适配问题

### DISCUSSION THREAD

**Discussion by Gengar-Asuka** [InfpHub/Raptor#28] [General]
我当前的屏幕分辨率为2560*1600，于是Raptor的UI整体就显得非常小，我将其和文件管理窗口放一起对比
也不排除我没找到适配或UI尺寸设置的入口。
![QQ截图20240227225705](https://github.com/InfpHub/Raptor/assets/53888616/554afb9e-c76c-49c9-805c-f16949d483f3)

**Comment by InfpHub** [comment]
> 我当前的屏幕分辨率为2560*1600，于是Raptor的UI整体就显得非常小，我将其和文件管理窗口放一起对比
> 也不排除我没找到适配或UI尺寸设置的入口。
> ![QQ截图20240227225705](https://github.com/InfpHub/Raptor/assets/53888616/554afb9e-c76c-49c9-805c-f16949d483f3)
> 

近几天就会发布 2024.2.0 版本，将解决该问题。
